### ISSUE TITLE
Ревью темы 1

### ISSUE THREAD

**Issue by max-arshinov** [EduWebDotNet/Methodology#10]
Изучить ссылки темы №1: https://github.com/max-arshinov/web-dev-csharp-react/tree/main/hands-on/2/1/Homework1.
Все ли ссылки актуальны?
Есть ли лучшие/более актуальные источники?
Стоит ли изменить вопросы для самопроверки?

**Comment by MorozovKirill310** [comment]
1. [Программисты-оптимисты](https://habr.com/ru/post/178905/)
2. [Джедайская техника](https://www.youtube.com/watch?v=jt3_sq8LQYM)
3. [Память и обучение](https://www.youtube.com/watch?v=3_SVHS6t_LM)
4. [Матрица Эйзенхауэра](https://timeweb.com/ru/community/articles/chto-takoe-matrica-eyzenhauera)
5. [О пользе лаконичности](https://habr.com/ru/post/340434/)
6. [Юнит-тестирование для чайников](https://habr.com/ru/post/169381/)
7. [Ежедневная работа с Git](https://habr.com/ru/post/174467/)

> Все ли ссылки актуальны?

Да. Все ссылки актуальны.

> Есть ли лучшие/более актуальные источники?

Ответить на этот вопрос уже сложнее в силу нижеизложенного.

Материалы из первой темы мне показались сильно разрозненными.
Рассказано много, интересно, познавательно, но о разном, сильно разном.
Тут и память, и управление временем и умение оценивать и приоритизировать задачи, и unit-тестирование, и работа с git.
За этим "наступлением по площадям", как по мне, потерялся фокус, не ясна конечная цель.
Выстрелом в голову является практическое задание - консольный калькулятор. С теоретической частью я смог соотнести разве что требование покрытия тестами, о котором рассказано в материале из п. 6.

Смею предположить, что рассказать, поделиться с обучающимися чем-то подобным хочется, но сжатые сроки семестра не позволяют сделать это обстоятельно.

В рамках текущей активности (составление публичного онлайн курса), как я понимаю, мы не ограничены какими-то ни было методическими планами.

Поэтому я предлагаю реструктурировать материалы первой темы.

1. 4. 5. Я бы встроил в некий модуль про soft skills. 
6. 7. Тестирование и Git - достойны, по моему мнению, отдельных, самостоятельных модулей.
2. Лекцию Аси Казанцевой про память, полагаю, можно довольно удачно вписать в рамки классической университетской аудиторной программы. Но как (а главное зачем) нацелено встраивать ее в в рамки нашего онлайн курса мне сложно представить.
3. Не смог осилить из-за плохого звука.

**Comment by kulakovt** [comment]
Мы можем разделать работу над каждым модулем (уроком) на три этапа:

1. Проверка актуальности — быстро переносим существующий материал
2. Написание связующего материала — оформляем бездушный набор ссылок в привлекательный текст
3. Рефакторинг и улучшения — пересматриваем и изменяем модуль на основе экспертного опыта

Направление работы над курсом следует сосредоточить в ширину. Т.е. сделать сначала первый этап для всех модулей. Кажется что он самый лёгкий и предсказуемый. Затем вернуться и заняться вторым этапом. Ибо здесь уже нужны писательские задатки и не каждый это осилит. После завершения первых двух этапов, можно вступать в третий. Он кажется уже бесконечным и никогда не должен останавливаться. В процессе прохождения первых двух этапов, будем составлять в wiki предложения по улучшению (как в приведённом issue).

Для первого урока нужно сделать сразу первый и второй этап. Чтобы мы понимали на что ориентироваться. Возможно, для второго тоже, ибо он более показателен для всего остального материала чем первый. Безусловно первый урок нуждается в самом большом рефакторинге, относительно остальных. Может быть его стоит начать немного раньше чем закончится второй у всего курса.

**Comment by max-arshinov** [comment]
@MorozovKirill310 я согласен, что этот модуль нужно разбить. Сейчас тема существует в таком виде, потому что лично мне было удобно весь этот материал дать одним блоком, чтобы далее не отвлекаться на не связанные темы.

**Comment by max-arshinov** [comment]
@max-arshinov @ka_morozov тут предложение, как разделить первую тему

**Comment by max-arshinov** [comment]
@Ubloobok, у тебя точно есть какие-то ссылки про нейробиологию

**Comment by max-arshinov** [comment]
@max-arshinov @Ubloobok

**Comment by Ubloobok** [comment]
Что касается модуля "учимся учиться" в целом, выглядит будто то ли его надо давать кусочками среди прочих материалов, то ли давать людям комплекс теории+практики для прохождения параллельно с другими модулями, т.к. невозможно конечно же сразу применить все те советы которые там упоминаются по самоорганизации. Да что там применить, даже осознать.

Что касается теории, 
Есть хороший цикл статей от Макса Коткова на Хабре, который на обывательском уровне вплетает Нейробиологию в Обучение.

+ Макс Котков. Пластичность мозга: путь к продуктивности или к выгоранию? https://habr.com/ru/company/oleg-bunin/blog/586220/
+ Макс Котков. Интеллектуальная производительность без выгорания: 7 техник использования мозга.
+ Часть 1. https://habr.com/ru/company/oleg-bunin/blog/587222/
+ Часть 2. https://habr.com/ru/company/oleg-bunin/blog/588911/ 
* Ограничивать количество рабочих часов.
* Разминка мозга 10-15 прежде чем приступить к трудному.
* Техника Гоголя - начни писать хоть что-нибудь.
* Техника Аватара - представляйте себя в идеальном состоянии.
* Заминка и Выгрузка - тоннель и ритуалы.
* Расслабляемся когда чувствуем напряжение и излишний фокус.
* Планировать и работу, и отдых.
